static/js/controllers/inbox.js

```diff
@@ -202,7 +212,11 @@ require('moment/locales');
         });
       };
 
-      $scope.$on('ContactUpdated', function() {
+      $scope.$on('ContactUpdated', function(contact) {
+        if (!contact || contact._deleted) {
+          // allow the ContactCtrl to handle contact deletion
+          return;
+        }
```

Comment:
Yep, feels v. low priority, unless end users likely to see significant failure rates or inconsistent response times with their Ajax requests.